src/pages/enterprise/auth-connect/azure-ad.md

```diff
@@ -0,0 +1,116 @@
+# Azure Active Directory Configuration
+
+## Configuration Details
+
+### Azure Configuration
+
+Before integrating Auth Connect into your Ionic app, you’ll need to get Azure Active Directory (AD) up and running.
+
+> Note: For complete information on configuring Azure AD, consult the [official B2C documentation](https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-create-tenant) which includes tutorials on creating a B2C tenant, registering applications, and more.
+
+#### Create an Azure AD B2C Tenant
+
+If you don't have one, [create a new B2C tenant](https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-create-tenant).
+
+#### Register an Application
+
+Sign into the [Azure Portal](https://portal.azure.com) then navigate to the `Azure AD B2C` service page (the easiest way to find it is to search for "b2c", then choose "Azure AD B2C".)
+
+Begin by creating a new Application under Manage -> Applications -> Add. 
+
+![Azure app configuration settings](/docs/assets/img/native/azure-app-settings.png)
+
+Give your app a new name, then toggle `Yes` for both _Web App_ and _Allow implicit flow_. For _Reply URL_, specify `http://localhost:8100/` along with the name of your app's core login page (typically, `login`).
+
+Next, toggle `Yes` for _Native client_. Note the _Redirect URIs_ that are displayed. 
+
+Next, choose your globally unique App Id, which is used both in the Azure configuration as well as Cordova/Capacitor. Typically, this takes the form of `company-AppName` or reverse DNS style - `com.company.app`.
+
+With that in hand, set the _Custom Redirect URI_. After the app user signs into Azure AD, this tells Auth Connect which page to redirect to in your app. Use the formula “uniqueId://page”, such as `com.company.app://callback`.
+
+After filling in all details above, click the Create button. 
+
+#### Create User Flows (Policies)
+
+Create at least one [User Flow](https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-create-user-flows), the series of pages your users will interact with to sign up/sign into their account. Note the `Name` of the User Flow you create, as it's needed for configuring the Auth Connect `Discovery URL` property.
+
+Azure AD is now ready to use with Auth Connect.
+
```

Comment:
might comment on noting the discovery url while creating the flow(s).
also, the discovery url for both password reset and create/login should both be noted.

Reply:
good call, added. thanks!